concepts/review.md

```diff
@@ -1,5 +1,103 @@
-# Work in Progress
+# Getting Analysis Review
+
+The current process for getting analysis review has not been optimized.
+If you run into problems, feel free to ask #datapipeline
+
+## Table of Contents
+
+<!-- toc -->
+
+# Finding a Reviewer
+
+There are two major types of review for most analyses:
+**Data review** and **Methodology review**.
+
+A **data reviewer** has **expert understanding of the dataset** your analysis is based upon.
+During data review, your reviewer will try to identify
+any issues with your analysis that come from misapplying the data.
+For example, sampling issues or data collection oddities.
+
+**Methodology review** primarily covers **statistical concepts**.
+For example, if you're training regressions, forecasting data, or doing complex
+statistical tests, you should probably get Methodology review.
+Many (most?) analyses at Mozilla use simple aggregations and little to no statistical inference.
+These analyses only require data review.
+
+Accordingly, we suggest you **first find a data reviewer for your analysis**.
+Your data reviewer will tell you if you should get methodology review
+and will help you find a reviewer.
+
+## Data Reviewers
+
+To get review for your analysis,
+contact one of the data reviewers listed for the dataset your analysis is based upon:
+
+| Dataset           | Reviewers   |
+| :---              | :---        | 
+| Longitudinal      |             |
+| Cross Sectional   |             |
+| Main Summary      |             |
+| Crash Summary     |             |
+| Crash Aggregate   |             |
+| Events            |             |
+| Sync Summary      |             |
+| Addons            |             |
+| Client Count      |             |
+
+# Jupyter Notebooks
+
+It's difficult to review Jupyter notebooks on Github.
+The notebook is stored as a JSON object, with python code stored in strings.
+This makes **commenting on particular lines very difficult**.
+Because the output is stored next to the code,
+it's also very **difficult to review the diff** for a given notebook.
+
+## RTMO
+
+For simple notebooks, the best way to get review is through the
+[knowledge repo](https://github.com/mozilla/mozilla-reports).
```

Comment:
Step-by-step instructions on how to go from a runnning ipython notebook to a review request via RTMO would be very helpful here.